src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2410,7 +2410,7 @@ namespace winrt::TerminalApp::implementation
                                                                 IVectorView<Profile> profiles,
                                                                 IMapView<winrt::hstring, ColorScheme> schemes)
     {
-        IVector<SettingsLoadWarnings> warnings;
+        IVector<SettingsLoadWarnings> warnings{ winrt::single_threaded_vector<SettingsLoadWarnings>() };
```

Comment:
I hate that we're heap allocating a thing that in most instances will not be used at all, but it fixes a bug and I cannot die on the hill of purity.